spec/lib/tabular_form_builder_spec.rb

```diff
@@ -533,4 +533,93 @@
       end
     end
   end
+
+  # test the label that is generated for various field types
+  describe 'labels for fields' do
+    let(:options) { {} }
+    shared_examples_for "generated label" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.